# Plan goal.
sensor_msgs/JointState start_config
sensor_msgs/JointState goal_config
geometry_msgs/Pose goal_pose
std_msgs/Bool has_finger_joint
sensor_msgs/JointState finger_joint_config

int32 goal_type
int32 CONFIG_GOAL = 0
int32 POSE_GOAL   = 1
---
# Plan result
trajectory_msgs/JointTrajectory trajectory

int32 result
int32 SUCCESS = 0
int32 FAILURE = -1
---
# Plan feedback.
int32 placeholder
